thanos/charts/templates/ruler/ruler-svc.yaml

```diff
@@ -1,6 +1,4 @@
-# SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
-# SPDX-License-Identifier: Apache-2.0
-
+{{ if and (.Capabilities.APIVersions.Has "monitoring.coreos.com/v1") .Values.thanos.ruler.enabled (not .Values.thanos.query.standalone) }}
```

Comment:
Could you add some helm chart tests also please?

Reply:
good idea, added some test for when we disable the ruler and fixed service naming in tests too